pkgs/tools/misc/tmuxinator/default.nix

```diff
@@ -37,6 +37,12 @@ buildRubyGem rec {
 
   propagatedBuildInputs = [ erubis thor xdg ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion $GEM_HOME/gems/${gemName}-${version}/completion/tmuxinator.{bash,zsh,fish}
```

Comment:
Not 100% sure getting to gem source via `$GEM_HOME/gems/${gemName}-${version}/` is _the_ way to do it (decided to use this approach as I've seen it in other places)